pkg/config/config.go

```diff
@@ -893,6 +893,25 @@ func InitConfig(config Config) {
 	// Time in seconds
 	config.BindEnvAndSetDefault("logs_config.file_scan_period", 10.0)
 
+	// Controls how wildcard file log source are prioritized when there are more files
```

Comment:
Since we're adding a release note for this, would it make sense to add this to `config_template.yaml`? Maybe move the docs there as well, so that they are visible to users.